### PR TITLE
Added URL to all metadata containers

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: neptune/new/metadata_containers/metadata_container.py [hunk 1]
```diff
@@ -155,6 +155,11 @@ def _label(self) -> str:
     def _docs_url_stop(self) -> str:
         raise NotImplementedError
 
+    @property
+    # @abc.abstractmethod
```
Comment (pkasprzyk) [inline neptune/new/metadata_containers/metadata_container.py:159]: either uncomment or remove ;)

File: neptune/new/metadata_containers/metadata_container.py [hunk 2]
```diff
@@ -338,7 +343,14 @@ def _define_attribute(self, _path: List[str], _type: AttributeType):
     def _get_root_handler(self):
         return Handler(self, "")
 
+    def get_url(self) -> str:
+        """Returns the URL that can be accessed within the browser"""
+        return self._url
```
Comment (pkasprzyk) [inline neptune/new/metadata_containers/metadata_container.py:348]: I don't really see the advantage of this vs simply having abstract `get_url` and implementing it in subclasses. What am I missing?

File: neptune/new/metadata_containers/metadata_container.py (hunk 1 above)
Comment (Raalsky) [reply neptune/new/metadata_containers/metadata_container.py]: Sure, uncommented

File: neptune/new/metadata_containers/metadata_container.py (hunk 2 above)
Comment (pkasprzyk) [reply neptune/new/metadata_containers/metadata_container.py:348]: discussed: the advantage is defining docstring once :)

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**